_posts/2020-01-16-openfaas-flux.md

```diff
@@ -17,6 +17,10 @@ This is a step-by-step guide on how to set up a [GitOps](https://www.weave.works
 workflow for OpenFaaS on Kubernetes with [Flux](https://github.com/fluxcd/helm-operator) and its [Helm Operator](https://github.com/fluxcd/helm-operator).
 You'll be installing OpenFaaS and deploy functions on a Kubernetes cluster in a declarative manner, using only a GitHub repository and the git CLI.
 
+> Update: 2021-01-15
```

Comment:
Can we just change it in the post?

Reply:
Do you know what, when I looked it this yesterday I couldn't see any references for looking. I clearly needed a morning of woodworking and bees to enable me to see things again.

I've updated the GitHub repo behind this post